readthedocs/config/tests/test_config.py

```diff
@@ -212,7 +227,7 @@ def test_use_system_site_packages_defaults_to_false():
     build = get_build_config({'python': {}}, get_env_config())
     build.validate()
     # Default is False.
-    assert not build.use_system_site_packages
+    assert not build.python.use_system_site_packages
```

Comment:
What's the correct here? `use_system_site_packages` or `system_packages`?

It seems that we are using all the other names from V2 but this one only from V1. I think we should call it as in V2 (https://github.com/rtfd/readthedocs.org/pull/4451/files#diff-33bbde7fca852e881b51aa90f9f86b3fR83) `build.python.system_packages`

Reply:
`use_system_site_packages` is from the main interface. Yeah, `system_packages` is short

Reply:
So, it should be `use_system_packages` or just `system_packages`?